src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -38,6 +38,27 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
                 .anyMatch(preppedWord::equalsIgnoreCase);
     }
 
+    /**
+     * Returns true if the {@code sentence} contains {@code words}.
+     *   Ignores case, but a exact match is required.
+     *   <br>examples:<pre>
+     *       containsMultipleWordsIgnoreCase("ABc def", "abc def") == true
+     *       containsMultipleWordsIgnoreCase("ABc def", "abc DEF") == true
+     *       containsMultipleWordsIgnoreCase("ABc def", "ABc d") == false //not an exact match
+     *       </pre>
+     * @param sentence cannot be null
+     * @param words cannot be null, cannot be empty, can be multiple words
+     */
+    public static boolean containsMultipleWordsIgnoreCase(String sentence, String words) {
+        requireNonNull(sentence);
+        requireNonNull(words);
+
+        String preppedWord = words.trim();
+        checkArgument(!preppedWord.isEmpty(), "Word parameter cannot be empty");
+
+        return sentence.equalsIgnoreCase(preppedWord);
+    }
```

Comment:
Would be better to name it as "exactlyMultipleWordsIgnoreCase"? "contains" is a little misleading.